hilti/runtime/src/tests/regexp.cc

```diff
@@ -70,6 +70,11 @@ TEST_CASE("match") {
         CHECK_EQ(RegExp("ab+c", regexp::Flags{.use_std = 1}).match("abbbcdef"_b), 1);
         CHECK_EQ(RegExp("ab+c", regexp::Flags{.use_std = 1}).match("012abbbc345"_b), 0);
     }
+
```

Comment:
We still need to update the revision we bump the justrx submodule to after https://github.com/rsmmr/justrx/pull/2 is merged.